plugins/chat/spec/system/transcript_spec.rb

```diff
@@ -0,0 +1,246 @@
+# frozen_string_literal: true
+
+RSpec.describe "Quoting chat message transcripts", type: :system, js: true do
+  fab!(:current_user) { Fabricate(:user) }
+  fab!(:chat_channel_1) { Fabricate(:chat_channel) }
+  fab!(:chat_message_1) { Fabricate(:chat_message, chat_channel: chat_channel_1) }
+  fab!(:chat_message_2) { Fabricate(:chat_message, chat_channel: chat_channel_1) }
+  fab!(:chat_message_3) { Fabricate(:chat_message, chat_channel: chat_channel_1) }
+  fab!(:chat_message_4) { Fabricate(:chat_message, chat_channel: chat_channel_1) }
+  fab!(:chat_message_5) { Fabricate(:chat_message, chat_channel: chat_channel_1) }
+  fab!(:topic) { Fabricate(:with_posts_topic) }
+  let(:last_topic_post) { topic.posts.last }
+
+  let(:chat_page) { PageObjects::Pages::Chat.new }
+  let(:chat_channel_page) { PageObjects::Pages::ChatChannel.new }
+  let(:topic_page) { PageObjects::Pages::Topic.new }
+
+  before do
+    chat_system_bootstrap(current_user, [chat_channel_1])
+    sign_in(current_user)
+  end
+
+  def select_message(message, mobile: false)
```

Comment:
Candidate to move into a helper/page object after Joffrey's PR.

Reply:
you don't need mobile anymore. You can use `mobile: true` at the example level.

I see you struggle in trying to have something to work for both mobile/desktop, I don't think we should try to do this. If this is different screens we should have different tests, otherwise it becomes a nightmare maintenance each time you chance things in a screen, you have two screens and helpers to change.

Reply:
I am not sure what you mean by this, I did use it at the example level. I guess I could just split this into `select_message_desktop` and `select_message_mobile` instead so we don't have the conditions